hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/Retransmitter.java

```diff
@@ -38,7 +39,8 @@ public void reloadOffsets(SubscriptionName subscriptionName, Consumer consumer)
 
             for (PartitionOffset partitionOffset : offsets) {
                 SubscriptionPartitionOffset subscriptionPartitionOffset = new SubscriptionPartitionOffset(
```

Comment:
What do you think about passing `partitionOffset` to `KafkaConsumerOffsetMover` instead `subscriptionPartitionOffset`? If access to subscription is necessary in `KafkaConsumerOffsetMover` we can pass it through the class constructor.